### PR TITLE
Fixed autoinvite.lua

### PR DESCRIPTION
**Pull request by Petewilmer**
isin() deprecated so replaced with table.contains() or == where appropriate

### CODE REVIEW

File: addons/autoinvite/autoinvite.lua [hunk 1]
```diff
@@ -131,6 +131,7 @@ function add_item(mode, ...)
 	local new = names - settings[mode]
 	if not new:empty() then
 		settings[mode] = settings[mode] + new
+		log(new:format()..'test')
```
Comment (z16) [inline addons/autoinvite/autoinvite.lua]: Is this meant to be there?

File: addons/autoinvite/autoinvite.lua [hunk 2]
```diff
@@ -158,11 +159,10 @@ end
 windower.register_event('addon command',function (command, ...)
 	command = command and command:lower() or 'status'
 	local args = T{...}
-	
 	-- Changes whitelist/blacklist mode
 	if command == 'mode' then
 		local mode = args[1] or 'status'
-		if mode:isin(aliases:keyset()) then
+		if table.contains(aliases, mode) then
```
Comment (z16) [inline addons/autoinvite/autoinvite.lua]: This should either be `table.contains(aliases:keyset(), mode)` or `aliases:keyset():contains(mode)` (preferably the latter).

File: addons/autoinvite/autoinvite.lua [hunk 3]
```diff
@@ -176,10 +176,10 @@ windower.register_event('addon command',function (command, ...)
 	elseif command == 'tellback' then
 		status = args[1] or 'status'
 		status = string.lower(status)
-		if status:isin(on) then
+		if status == 'on' then
```
Comment (z16) [inline addons/autoinvite/autoinvite.lua]: This needs to be a table check as well: `on:contains(status)`. Same for `off` below

File: addons/autoinvite/autoinvite.lua [hunk 4]
```diff
@@ -189,15 +189,15 @@ windower.register_event('addon command',function (command, ...)
 			return
 		end
 		
-	elseif command:isin(aliases:keyset()) then
+	elseif table.contains(aliases, command) then
```
Comment (z16) [inline addons/autoinvite/autoinvite.lua]: Same as for the `:keyset()` above.

File: addons/autoinvite/autoinvite.lua [hunk 5]
```diff
@@ -189,15 +189,15 @@ windower.register_event('addon command',function (command, ...)
...
 		mode = aliases[command]
 		names = args:slice(2):map(string.ucfirst..string.lower)
 		if args:empty() then
 			log(mode:ucfirst()..':', settings[mode]:format('csv'))
 		else
-			if args[1]:isin(addstrs) then
+			if table.contains(addstrs, args[1]) then
```
Comment (z16) [inline addons/autoinvite/autoinvite.lua]: Might wanna change these to `addstrs:contains(args[1])`, although that's not necessary. Same below.

File: addons/autoinvite/autoinvite.lua (hunk 1 above)
Comment (Petewilmer) [inline addons/autoinvite/autoinvite.lua]: Nope, my bad, removed from latest version

File: addons/autoinvite/autoinvite.lua (hunk 2 above)
Comment (Petewilmer) [inline addons/autoinvite/autoinvite.lua]: Still getting used to lua, never used it before, changed as suggested, not using keyset() was silly mistake on my part though. Changed in latest version

File: addons/autoinvite/autoinvite.lua (hunk 3 above)
Comment (Petewilmer) [inline addons/autoinvite/autoinvite.lua]: Changed in latest version

File: addons/autoinvite/autoinvite.lua (hunk 4 above)
Comment (Petewilmer) [inline addons/autoinvite/autoinvite.lua]: Changed in latest version

File: addons/autoinvite/autoinvite.lua (hunk 5 above)
Comment (Petewilmer) [inline addons/autoinvite/autoinvite.lua]: Changed in latest version

**Comment by z16** [comment]
There's still a `:keyset()` missing at the second `aliases` you changed.

**Comment by Petewilmer** [comment]
Updated, seems I need more coffee...

**Pull request merged**